tests/models/onnxModels/castToFloat.onnxtxt

```diff
@@ -0,0 +1,63 @@
+ir_version: 3
+producer_name: "test4glow"
+opset_import { 
+  version: 7
+}
+
+# ONNX TensorProto.DataType:
+#    UNDEFINED = 0;
+#    FLOAT = 1;
+#    UINT8 = 2;
+#    INT8 = 3;
+#    UINT16 = 4;
+#    INT16 = 5;
+#    INT32 = 6;
+#    INT64 = 7;
+#    STRING = 8;
+#    BOOL = 9;
+#    FLOAT16 = 10;
+#    DOUBLE = 11;
+#    UINT32 = 12;
+#    UINT64 = 13;
+#    COMPLEX64 = 14;
+#    COMPLEX128 = 15;
+
+graph {
+  name: "test-model"
+  node {
+    input: "data"
+    output: "toFloat"
+    name: "toFloat"
+    op_type: "Cast"
+    attribute {
+      name: "to"
+      i: 1
+      type: INT
+    }
+  }
+  input {
+    name: "data"
+    type {
+      tensor_type {
+        elem_type: 1
```

Comment:
Since we're casting to float, can you make this elem_type non-float?

Reply:
Done